crates/task-impls/src/view_sync.rs

```diff
@@ -470,9 +470,12 @@ impl<
                 }
 
                 self.num_timeouts_tracked += 1;
+                let leader = self.membership.get_leader(view_number);
                 error!(
-                    "Num timeouts tracked since last view change is {}. View {} timed out",
-                    self.num_timeouts_tracked, *view_number
+                    %leader,
```

Comment:
Would it be nicer to use `cdn_proto::mnemonic(leader)`? 

This was a function Rob used to display names in the CDN brokers; I used this in the network task trace too

Reply:
I think it would be good to do both, in the sequencer logs and metrics we have the pub key itself